app/config/errors.php

```diff
@@ -534,6 +534,11 @@
         'description' => 'A Domain with the requested ID already exists.',
         'code' => 409,
     ],
+    Exception::DOMAIN_VERIFICATION_FAILED => [
```

Comment:
This should be `DOMAIN_VARIABLE_INCORRECT`

```suggestion
    Exception::DOMAIN_VARIABLE_INCORRECT => [
```

I may have been unclear earlier when I mentioned the placement of this error. I meant it should go under the:

```
    Exception::DOMAIN_VERIFICATION_FAILED => [
        'name' => Exception::DOMAIN_VERIFICATION_FAILED,
        'description' => 'Domain verification for the requested domain has failed.',
        'code' => 401,
    ],
```

block some lines below.